datamade/ccfp-asset-dashboard
# Table bug fix

## Overview
Fixes a bug detailed in #34. This PR closes that issue.

## Testing Instructions
- Visit the review app's home page.
- Don't add a project.
- See the list table where it says "No data available in table"
- Click and make sure it doesn't take you to another page
